spec/lib/blog_fetcher_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe BlogFetcher do
+ describe ".feed" do
+    context "nothing cached" do
+      it "fetches requested feed" do
+        redis = Redis.new
+        expect(RestClient).to receive(:get)
+          .with(BlogFetcher::BASE_URL + BlogFetcher::ENDPOINTS[:recent])
+        BlogFetcher.feed(:recent)
+        expect(redis.ttl("#{BlogFetcher::KEY_BASE}:recent"))
+          .to be_within(1).of(BlogFetcher::EXPIRE_SECONDS)
+      end
+    end
+
+    context "feed already cached" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.